sriov/sriov.go

```diff
@@ -82,6 +82,15 @@ func moveIfToNetns(ifname string, netns ns.NetNS) error {
 		return fmt.Errorf("failed to lookup vf device %v: %q", ifname, err)
 	}
 
+	if err = netlink.LinkSetDown(vfDev); err != nil {
+		return fmt.Errorf("failed to down vf device %q: %v", ifname, err)
+	}
+	index := vfDev.Attrs().Index
+	vfName := fmt.Sprintf("dev%d", index)
+	if renameLink(ifname, vfName); err != nil {
+		return fmt.Errorf("failed to rename vf %v to %v: %q", ifname, vfName, err)
```

Comment:
``return fmt.Errorf("failed to rename vf device %q to %q: %v, ifname, vfName, err)``